enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -0,0 +1,37 @@
+import { merge } from "icepick";
+
+import type { MetabaseComponentTheme } from "embedding-sdk";
+
+/**
+ * Default theme options for Metabase components.
+ *
+ * While these theme options are primarily used by the React Embedding SDK
+ * to provide extra customization for SDK users,
+ * the options below are used to provide default values to components
+ * such as charts, data tables and popovers.
+ */
+export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
+  table: {
+    cell: {
+      textColor: "text-brand",
```

Comment:
is `text-brand` really a thing? I see we have some references in the current codebase, but this isn't defined in [`palette.ts`](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/lib/colors/palette.ts#L9)

Reply:
@WiNloSt Good catch! Just realized that this is probably a no-op. It was used for hover in HeaderCell, but I cannot find any references in the code. Let me remove this.

```
export const HeaderCell = styled.div`
  color: ${color("text-medium")};

  &:hover {
    color: ${color("text-brand")};
  }
`;
```